File: ambeth/Ambeth.Log/ambeth/config/LogConfigurationConstants.cs
using De.Osthus.Ambeth.Annotation;
using System;
using System.ServiceModel;

namespace De.Osthus.Ambeth.Log
{
    [ConfigurationConstants]
    public sealed class LogConfigurationConstants
    {
		/// <summary>
		/// The path to the file to which Ambeth should write the log statements. No default value, if not set Ambeth will not log to file.
		/// </summary>
        public const String LogFile = "ambeth.log.file";

        private LogConfigurationConstants()
        {
            // intended blank
        }
    }
}

File: jambeth/jambeth-log/src/main/java/de/osthus/ambeth/log/LogConfigurationConstants.java
package de.osthus.ambeth.log;

import de.osthus.ambeth.annotation.ConfigurationConstants;

@ConfigurationConstants
public class LogConfigurationConstants
{
	/**
	 * The path to the file to which Ambeth should write the log statements. No default value, if not set Ambeth will not log to file.
	 */
	public static final String LogFile = "ambeth.log.file";

	private LogConfigurationConstants()
	{
		// Intended blank
	}
}
